### PR TITLE
[16.0][ADD] project_update_visible

### PR DESCRIPTION
**Pull request by adasatorres**
This addon modifies the form view of the product.update model to make the project_id field visible.

### CODE REVIEW

File: project_update_visible/views/project_update_views.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<odoo>
+  <!-- model.name inherit form view -->
```
Comment (etobella) [inline project_update_visible/views/project_update_views.xml]: You don't need to keep this information

File: project_update_visible/views/project_update_views.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <record id="project_update" model="ir.ui.view">
+    <field name="name">project.update.inherit.view.form</field>
+    <field name="model">project.update</field>
+    <field name="inherit_id" ref="project.project_update_view_form" />
+    <field name="arch" type="xml">
+      <!-- Add your xpath here -->
```
Comment (etobella) [inline project_update_visible/views/project_update_views.xml]: You don't need to keep this information

File: project_update_visible/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Project Update Visible",
+    "summary": """
+        Visible project_id in project.update view form.
+    """,
+    "author": "Odoo Community Association (OCA)",
```
Comment (etobella) [inline project_update_visible/__manifest__.py]: Add Binhex as author:

```suggestion
    "author": "Binhex, Odoo Community Association (OCA)",
```

**Review by etobella — COMMENTED** [review]
In general, it is ok, but you should need to apply some nitpicking changes :smile:

File: project_update_visible/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (etobella) [inline project_update_visible/__manifest__.py:3]: Also, you need to add the copyright information on all python files (except __init__.py files)
```suggestion
# Copyright 2023 Binhex (YOUR WEBSITE)
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).

{
```

**Comment by adasatorres** [comment]
@etobella Thanks for your help, I have already made the changes.

File: project_update_visible/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    # Copyright 2023 Binhex (https://www.binhex.cloud)
```
Comment (etobella) [inline project_update_visible/__manifest__.py]: The copyright should be on top of the file

```python
# Copyright 2023 Binhex (https://www.binhex.cloud)
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
{
....
```

Check for example any other module from the repository :wink:

**Review by etobella — APPROVED** [review]
I am already approving it, as it is just a small comment that will be easily solved.

Remember to squash all your commits in a single one.

You should need `git rebase` to squash them :wink:

**Review by szalatyzuzanna — APPROVED** [review]

**Review by nramosdev — APPROVED** [review]

File: project_update_visible/readme/CONTRIBUTORS.rst [hunk 6]
```diff
@@ -0,0 +1,2 @@
+* `Binhex <https://www.binhex.cloud>`_:
+  * Adasat Torres de León
```
Comment (pedrobaeza) [inline project_update_visible/readme/CONTRIBUTORS.rst]: **issue:** For proper RST syntax, it should be:

```suggestion
* `Binhex <https://www.binhex.cloud>`_:

  * Adasat Torres de León
```

File: project_update_visible/readme/DESCRIPTION.rst [hunk 7]
```diff
@@ -0,0 +1 @@
+This addon allows see the project_id field in the form view of the project.update model.
```
Comment (pedrobaeza) [inline project_update_visible/readme/DESCRIPTION.rst]: **suggestion**: This description should be user oriented, not technical one. so I suggest to put:

```suggestion
This add-on allows to see the project for which the update is being done
directly in the same screen.
```

File: project_update_visible/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    "summary": """
+        Visible project_id in project.update view form.
+    """,
+    "author": "Binhex, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "category": "Project",
+    "version": "16.0.1.0.0",
+    "installable": True,
+    "license": "AGPL-3",
+    "depends": ["base", "project", "sale_timesheet"],
```
Comment (pedrobaeza) [inline project_update_visible/__manifest__.py]: **nitpick:** You don't need to put the whole dependency chain if you don't want. Here the Python zen of "better explicit than implicit" is very difficult to follow for this case, and maybe in future versions the dependencies change, and you are declaring here an extra module that is not needed at all.

```suggestion
    "depends": ["sale_timesheet"],
```

**Review by pedrobaeza — COMMENTED** [review]
Thank you for this first PR. You are on the road!

File: project_update_visible/__manifest__.py (hunk 8 above)
Comment (jelenapoblet) [reply project_update_visible/__manifest__.py]: Thanks for the help @pedrobaeza and @etobella you guys are awesome!

**Review by jelenapoblet — APPROVED** [review]

**Pull request merged**